src/Tests/Microsoft.NET.Pack.Tests/GivenThatWeWantToPackAHelloWorldProject.cs

```diff
@@ -85,19 +88,21 @@ public void It_fails_if_nobuild_was_requested_but_build_was_invoked()
                 .HaveStdOutContaining("NETSDK1085");
         }
 
-        [Fact]
-        public void It_packs_with_release_if_PackRelease_property_set()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void It_packs_with_release_if_PackRelease_property_set(bool optedOut)
         {
+            if (optedOut)
+            {
+                Environment.SetEnvironmentVariable(EnvironmentVariableNames.DISABLE_PUBLISH_AND_PACK_RELEASE, "true");
```

Comment:
We shouldn't be setting environment variables in the test process.  You can do the following instead:

```c#
            packCommand
                .WithEnvironmentVariable(EnvironmentVariableNames.DISABLE_PUBLISH_AND_PACK_RELEASE, "true")
                .Execute()
                .Should()
                .Pass();
```